base/docs/basedocs.jl

```diff
@@ -1288,9 +1288,10 @@ kw"Union{}"
 """
     Union{Types...}
 
-A type union is an abstract type which includes all instances of any of its argument types.
+A type union is an abstract type which includes all instances of any of its argument types. The empty
+union [`Union{}`](@ref) is the bottom type of julia.
```

Comment:
would capitalize this as it refers to the language properties, not the command-line executable